book/website/community-handbook/style/style-special-content.md

```diff
@@ -1,5 +1,5 @@
-(ch-style-more-features)=
-# Glossary, Special Blocks and More Styling
+(ch-style-special-content)=
+# Glossary and Special Content Blocks
 
 Jupyter Book offers more options for styling its chapters and create a more comprehensive book.
 In this subchapter, we have discussed a few more features that we recommend using in _The Turing Way_.
```

Comment:
First, merge the #1642 and pull changes to this branch - and then please go ahead and merge this PR whenever you are ready.